src/utils/helpers.js

```diff
@@ -29,3 +29,59 @@ export const createIdsSelector = createSelectorCreator(memoizeResult);
 
 // Use this selector when you want a shallow comparison of the arguments and you don't need to memoize the result
 export const createShallowSelector = createSelectorCreator(defaultMemoize, shallowEqual);
+
+// isMinimumServerVersion will return true if currentVersion is equal to higher or than the
+// the provided minimum version. A non-equal major version will ignore minor and dot
+// versions, and a non-equal minor version will ignore dot version.
+// currentVersion is a string, e.g '4.6.0'
+// minMajorVersion, minMinorVersion, minDotVersion are integers
+export function isMinimumServerVersion(currentVersion, minMajorVersion = 0, minMinorVersion = 0, minDotVersion = 0) {
+    if (!currentVersion || typeof currentVersion !== 'string') {
+        return false;
+    }
+
+    let firstDotIndex = currentVersion.indexOf('.');
+    if (firstDotIndex < 0) {
+        firstDotIndex = currentVersion.length;
+    }
+
+    let secondDotIndex = currentVersion.indexOf('.', firstDotIndex + 1);
```

Comment:
If the firstDotIndex < 0, couldn't we skip doing this check. If there wasn't 1 '.', there won't be two.